components/kubeflow/pytorch-launcher/src/launch_pytorchjob.py

```diff
@@ -29,16 +30,16 @@ def get_current_namespace():
 
 
 # Patch PyTorchJob APIs to align with k8s usage
-class V1PyTorchJob(V1PyTorchJob_original):
+class V1PyTorchJob(KubeflowOrgV1PyTorchJob):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.openapi_types = self.swagger_types
+        # self.openapi_types = self.swagger_types
```

Comment:
Why is this removed?

Reply:
I moved to importing KubeflowOrgV1PyTorchJob from kubeflow.training and this does not have a swagger_types attribute. With the previous `from kubeflow.pytorchjob import V1PyTorchJob` there was an error around using floats and ints within the numpy config file.
```
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
```

Reply:
I've removed these functions altogether as they are no longer required.